_data/projects/CASE.yml

```
@@ -0,0 +1,14 @@
+name: CASE
+desc: A frameworkless and Open Source tool to build CRUD applications as dashboards or admin panel.
+site: https://case.app
+tags:
+  - appbuilder
+  - typescript
+  - adminpanel
+  - CRUD
```

All tags need to be lowercase to be consistent

```suggestion
  - crud
```